replacy/scorer.py

```diff
@@ -79,16 +79,13 @@ def __call__(self, segment, score_type="perplexity"):
             bos = text.capitalize() == text
             eos = text[-1] in string.punctuation
 
-        # log10 prob
-        score = self.model.score(text, bos=bos, eos=eos)
 
         if score_type == "log":
+            # log10 prob
+            score = self.model.score(text, bos=bos, eos=eos)
             return score
-
         elif score_type == "perplexity":
-            prob = 10.0 ** (score)
-            return prob ** (-1 / word_count)
-
+            return self.model.perplexity(text)
```

Comment:
why? :sob:

Reply:
I don't understand why we need to reimplement a calculation that the model we are using provides, and call it the same thing, but do it differently